.github/workflows/ci_cd.yml

```diff
@@ -152,6 +152,8 @@ jobs:
 
           echo "stable_container=${{ env.DOCKER_PACKAGE }}:${{ env.LATEST_STABLE_DOCKER_IMAGE_VERSION }}" >> $GITHUB_OUTPUT
 
+          cat $GITHUB_OUTPUT
```

Comment:
I'd avoid doing a cat of the whole GITHUB OUTPUT. Is this only for dev purposes ok your side @dipinknair ? If so, can we delete the cat commands?

Reply:
Yes. removed !